src/swagger.cr

```diff
@@ -4,9 +4,6 @@ require "./swagger/*"
 module Swagger
   VERSION = "0.1.1"
 
```

Comment:
Why delete the constant and place into the corner of code?

Reply:
As the commit says, prevent having circular dependencies.
Why the document module needs all other modules, just for this particular version?
Another way to solve it is to have a `version.cr`

Reply:
Doing this in another PR.